index.html

```diff
@@ -1401,6 +1406,8 @@ <h3>Types</h3>
               <td>
 A valid terms of use <a>type</a>. For example,<br>
 <code>"type": "OdrlPolicy2017"</code>)
+
+<p class="issue" data-number="1123"></p>
```

Comment:
Not sure that I'm a fan of adding issue markers throughout the spec. We have 65+ issues where we'd have to do that, only to go back through and remove them. It's unnecessary churn for the Editors (without good reason for the churn), and this isn't being done consistently, AFAICT, from the latest issue marker additions.

Reply:
It seems to me that issue markers are appropriate for significant publication milestones, like CRs. Less so, though arguable, for FPWDs. Far less so for ED-per-commit and/or daily-ED, though editor's prerogative probably holds.

Given that, I'd suggest that there be some rule of thumb that issue markers be added based on exceeding some number of comments and/or duration of discussion (I don't have a suggestion for either trigger), on the presumption that the issue is less likely to be closed quickly, and so more likely to persist beyond the next milestone publication and/or transition, which is likely to bring more unfamiliar readers for whom the issue markers are intended.